data-access-analysis-tools/index-rsp-warnings.rst

```diff
@@ -12,98 +12,60 @@
+You should only access our Butler repositories using standard Butler APIs provided in the Rubin software stack. This is both to prevent accidents but also to fulfil your role in helping us evaluate our software.
 
+RSP/IDF: The Missing Features
+------------------------------
 
+Many more features are on our roadmap for the RSP and its related services (Qserv, Butler, etc) and we have requirements to deliver them by the start of the survey. Some major ones to look forward in the future are:
 
-RSP Aspect Functionality Limits
--------------------------------
-
-Some major usability features will not be available, such as support for user database tables; support for parallelized or batch computation; the ability to sync files between RSP accounts and personal devices; and the ability to manage the sharing of data within private groups.
-
-Users installing code: DP0 delegates will be able to add software that can be installed without privileges into their own homespace (eg. via pip install --user) for their personal use to the extent that their quota allows.
-
-
-Bulk Downloads are not Supported
---------------------------------
-
-Bulk downloads: No, bulk data downloads from the RSP specifically, or the IDF in general, are not supported during DP0. This is because bulk download services are not yet in production, and because the major goals of DP0 are to allow Rubin Observatory to evaluate how the community uses the services, and for the community to become familiar with the RSP environment. Note that DESC has made the DC2 DR6 catalogs publicly available via: https://lsstdesc-portal.nersc.gov/.
-
-
-
-Stuff That We Know Does Not Work
---------------------------------
-
-*(Here is space for a list of known issues and stuff that doesn’t work in , e.g., the portal. Especially the portal. To avoid people thinking “This is too hard to figure out” when really it’s just a feature that’s not done.)*
-
-ADQL functionality that does not yet work: 
-AREA
-BOX
-COORDSYS
-COORD1
-COORD2
-INTERSECTS
-
-The only coordinate system currently supported is 'ICRS'.
+- Notebooks: We are planning on providing a filespace that you can access from your personal device (eg laptop) so you can use your favourite editor/IDE to write code in your local environment and save it in a way that makes it visible in your RSP notebook environment
```

Comment:
"e.g.,"

What does "we are planning" mean?  To be deployed *during* DP0.1?

Reply:
Uh I assumed missing features was understood to be missing from DP0. I'll adjust the phrasing.